### ISSUE TITLE
Thoughts on pagination?

### ISSUE THREAD

**Issue by mattfysh** [Sukairo-02/drizzle-graphql#24]
Is this something planned for drizzle-graphql? I noticed a couple of issues related to filtering, would pagination fall under that wide umbrella?

**Comment by Sukairo-02** [comment]
Offset  & limit are already supported in query params

**Comment by mdugue** [comment]
@Sukairo-02 what about cursor based pagination, relay compliancy etc. Any thoughts/ plans on that?

**Comment by Vincz** [comment]
Project is really nice, but we really need something for pagination. The minimum would be to return the total number of records so we can do simple pagination. But it would involve to add an intermediate payload for list queries. Is it something you could be open about?

**Comment by fenicento** [comment]
> Project is really nice, but we really need something for pagination. The minimum would be to return the total number of records so we can do simple pagination. But it would involve to add an intermediate payload for list queries. Is it something you could be open about?

agreed, this is really important. May I add, is this project actively maintained? I see no activity and I should understand if it's wise to keep using it for production.

**Comment by marcpalm** [comment]
We currently have an internal (pretty naive) implementation for Relay-compliant pagination based on this project, essentially sorting and selecting all elements, and then slicing it. Works pretty well for small tables, but we will soon have to optimize the query for larger tables with separate queries for totalCount and the returned elements (hasNext and hasPrevious can be reached by overfetching +1/-1). 

Our team also wonders whether this project going to be actively maintained (also going forward with v1+) and whether we could contribute back to the project?

**Comment by santigp258** [comment]
> We currently have an internal (pretty naive) implementation for Relay-compliant pagination based on this project, essentially sorting and selecting all elements, and then slicing it. Works pretty well for small tables, but we will soon have to optimize the query for larger tables with separate queries for totalCount and the returned elements (hasNext and hasPrevious can be reached by overfetching +1/-1).
> 
> Our team also wonders whether this project going to be actively maintained (also going forward with v1+) and whether we could contribute back to the project?

Hi, Marc. Do you have any plans to open source that?
